src/rtextures.c

```diff
@@ -2082,6 +2082,132 @@ void ImageBlurGaussian(Image *image, int blurSize) {
     ImageFormat(image, format);
 }
 
+// The kernel matrix is assumed to be square. Only supply the width of the kernel.
+void ImageKernelConvolution(Image *image, float* karnel, int karenlWidth){
+
+    if ((image->data == NULL) || (image->width == 0) || (image->height == 0) || karnel == NULL) return;
+
+    ImageAlphaPremultiply(image);
```

Comment:
Why alpha should be pre-multiplied for input image? Is not possible to apply the kernel otherwise?